.github/workflows/lts-test.yaml

```diff
@@ -0,0 +1,51 @@
+on:
+  # This should run for only pull requests for this repository's LTS branches
+  pull_request:
+    branches:
+      - '*-sp'
+name: LTS Library Test
+jobs:
+  lts-library-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # Only the downstream of this libraries are needed
+        repository:
+          - beam
+          # - google-api-java-client This does not work for LTS BOM Test due to bintray decommission
+          - google-auth-library-java
+          - google-maps-services-java
+          - grpc-java
+          - java-bigquery
+          - java-bigtable-hbase
+          - java-monitoring
+          - java-pubsub
+          - java-spanner
+          - java-storage
+          - java-trace
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - id: find_library_version
+        name: Find the library version in this branch
+        run: |
+          sudo apt-get install libxml2-utils
+          VERSION=`sed -e 's/xmlns="[^\s\\]*"//' pom.xml | xmllint --xpath '/project/version/text()' -`
+          echo "::set-output name=version::${VERSION}"
```

Comment:
@elharo (a followup from yesterday's discussion) This command extracts the version from the root pom.xml. This configuration file no longer needs to specify a hard-coded version. The value is then used at the end of this file as `version_override_value` parameter.

(I tweaked the command from https://github.com/googleapis/java-cloud-bom/blob/8a45813a6590d3a25dcaad4fa5d6beaeb341b3d3/.kokoro/bom-upper-bounds-check.sh#L43)

This sed+xmllint approach should work for Yoshi repositories. Other projects, such as [grpc-java](https://github.com/grpc/grpc-java/blob/master/build.gradle#L21) and [Beam](https://github.com/apache/beam/blob/master/gradle.properties#L27), will have other regular expression to extract their versions.